src/main/java/frc/robot/commands/AutonomousCommand.java

```diff
@@ -12,40 +12,199 @@
 
 package frc.robot.commands;
 
+import java.util.HashMap;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.auto.PIDConstants;
+import com.pathplanner.lib.auto.RamseteAutoBuilder;
+
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.subsystems.Claw;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.MagicArm;
 
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 
-/**
- *
- */
 public class AutonomousCommand extends CommandBase {
 
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
- 
+
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
 
+    private Command m_autonomousCommand;
+
+    private double ksVolts;
+    private double kvVoltSecondsPerMeter;
+    private double kaVoltSecondsSquaredPerMeter;
+    private double kPDriveVel;
+    private double kIDriveVel;
+    private double kDDriveVel;
+    private double kMaxSpeedMetersPerSecond;
+    private double kMaxAccelerationMetersPerSecondSquared;
+
+    private final Drivetrain m_driveTrain;
+    private final MagicArm m_arm;
+    private final Claw m_claw;
+
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
 
-    public AutonomousCommand() {
+    public AutonomousCommand(Drivetrain subsystem, MagicArm subsystem2, Claw subsystem3) {
```

Comment:
I don't think robotbuilder can possibly add this change, so if this is manual, make sure robotbuilder doesn't include the autonomous command in it's scope. Otherwise all of this will be overridden.